### DISCUSSION TITLE
How to Github Brain Badges

### DISCUSSION THREAD

**Discussion by NormalizerXD** [community/community#133702] [New to GitHub]
How to Github Brain Badges

**Comment by Vaibhav-kesarwani** [comment]
@NormalizerXD,  to get the Galaxy Brain achievement badge, you need to fulfill the following criteria:

Discussions should be in a public repository
1. Get marked as answers to 2 or more questions
2. The questions cannot be asked by you. In short, you cannot answer your own questions, those do not count.
3. In fact, answering your own questions could prevent the achievement from displaying in your profile page.
4. My advice here is to perhaps scour a few discussion forums, and answer a few questions based on your own knowledge/expertise. 5. Perhaps if someone finds your answer helpful, he/she would mark your answer and thus help you to unlock the badge.

Hope these steps help you in achieving your badge! Good luck :)

**Comment by rc-github-84** [comment]
how to link github vscode

**Reply by Vaibhav-kesarwani** [reply]
@rc-github-84 Simply sign in to Vscode with a GitHub account.

**Reply by anasahmedx5** [reply]
sign in to vscode with you github

**Comment by ghostinhershell** [comment]
Thanks so much for posting in our Community @NormalizerXD. Just wanted to chime in here and share that, because of an influx of users creating low quality, nonsensical or spam posts in the Community with the aim of earning certain Achievements, we’ve made the difficult decision to disable the ability to earn Achievements in our Community.

Note that this change does not affect your ability to earn badges in other projects and communities across GitHub. This change only applies to GitHub’s Community Discussions.

You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Let us know if you have any other questions 🚀

**Comment by Shreyash-SP80** [comment]
Hi everyone — sharing a quick note on GitHub profile badges (like the "Galaxy Brain") and how to earn them:

1) What these badges are
- Badges / Achievements on GitHub are awarded automatically by GitHub for certain actions (e.g., answering discussions, merged PRs, sponsorships, Arctic Code Vault, etc.). You can't add them manually to your profile — you have to earn them.

2) How to earn the "Galaxy Brain" badge
- Participate in public repository Discussions and post helpful answers.
- When one of your answers is marked "accepted" (or receives the required number of accepted answers across repositories), GitHub awards the Galaxy Brain achievement tier(s).

3) Other common badges (examples)
- Pull Shark — opened pull requests that were merged.
- Arctic Code Vault / Mars 2020 — special historical archive badges from GitHub programs.
- Sponsor / Public Sponsor — for sponsoring open-source work or being sponsored.

4) Quick tips to earn more achievements (ethically)
- Answer clearly and helpfully in Discussions (aim for accepted answers).
- Open quality pull requests and follow contribution guidelines.
- Sponsor or support open-source projects you care about.

5) Community rules to remember
- Don’t spam the Community for badges. Provide real value — helpful answers and well-reviewed PRs.
- This thread is for showcasing profiles (screenshots + short descriptions) — keep it community-friendly and non-promotional.

If you want, I can:
- Help you draft a short profile caption to paste with your screenshot.
- Suggest which parts of your profile to screenshot (pinned repos, README highlights).

Cheers!

**Comment by BhagyaAnand** [comment]
get 2 of your questions accepted

**Comment by gugs881** [comment]
To unlock this achievement, you need to:

Answer at least two discussions in public repositories.

Ensure your answers are marked as "accepted" by someone else (not yourself).

Avoid answering your own questions, as self-marking may prevent the badge from displaying.
Tiers depend on total accepted answers:

2 = Default

8 = Bronze

16 = Silver

32 = Gold

**Comment by tushartime** [comment]
simple and short -- getting your answer accepted on community discussion questions 
      you can also accept this and give me badge

**Comment by aljonard-dc** [comment]
nice

**Comment by mubtashimrafid** [comment]
Brain Badges are achievement badges you earn by helping people in Discussions.

The main “brain” achievement is Galaxy Brain.

You can get it by:

Joining repositories that have Discussions enabled. 
Answering questions in the Q&A / Question category.
Having your reply marked as the Accepted Answer by the person who asked.
Earning multiple accepted answers to unlock higher tiers.

Typical progression is:

Pair Extraordinaire → first accepted answer
Galaxy Brain → multiple accepted answers over time
Higher levels unlock as you keep helping people

A good strategy is to answer beginner questions clearly and quickly — helpful, simple replies get marked more often.

**Comment by Gecko51** [comment]
One thing worth clarifying since it gets buried in the thread: GitHub Community Discussions specifically has Achievements disabled. That was announced back in 2024 because of spam. So answering questions here will not earn the Galaxy Brain badge, no matter how many get accepted.

For other public repos, the badge still works. The criteria:

- Answer questions in the Discussions section of any public repo (must use the Q&A / Question category)
- Your answer needs to be marked as "Accepted" by the person who asked, not by yourself
- 2 accepted answers across public repos = first tier (Galaxy Brain shows up on your profile)
- Higher tiers unlock at 8, 16, and 32 accepted answers

If you want to earn it, look for repos that actively use GitHub Discussions for support -- open source frameworks, documentation projects, and tools with active communities usually have Q&A threads where people actually mark answers. Search GitHub for repos using discussions and filter by "unanswered" questions in categories you know well.

**Comment by MomoJasper22** [comment]
Hi Github people, here is how to get the GitHub Galaxy Brain badge:

1. Find a project: Go to any public repository on GitHub that has a Discussions tab ( or directly go to github.com/orgs/community/discussions ).
2. Look for Q&A: Open the discussions and find a post where someone is asking for help.
3. Write a helpful reply: Post a clear solution or code fix to answer their question.
4. Get marked as the answer: The author of the post (or repository owner) must click Mark as answer on your reply.
5 Unlock the badge: Get 2 accepted answers in repositories you don't own, and the badge will automatically appear on your profile!

**Comment by Hilal-11** [comment]
> How to Github Brain Badges

To get Galaxy Brain, your answer needs to be marked as the accepted answer in a Q&A-category GitHub Discussion.

Find a repo with Discussions enabled → go to the Q&A category (not General/Ideas).
Find an unanswered question and post a genuinely helpful answer.
The OP (or a maintainer) clicks "Mark as answer" on your reply.
Tiers (Bronze/Silver/Gold) unlock as you get more accepted answers.
